e2e_tests/test_prompt_gateway.py

```diff
@@ -260,3 +260,40 @@ def test_prompt_gateway_default_target(stream):
             response_json.get("choices")[0]["message"]["content"]
             == "I can help you with weather forecast or insurance claim details"
         )
+
+
+@pytest.mark.parametrize("prefill_enabled", [True, False])
```

Comment:
you may also want to add stream [true, false] to make sure this feature works in stream and non-stream case

Reply:
Prefill enabled will make stream = True

Reply:
I mean steam for response from upstream. I do think it would have no effect on it as when we engage with parameter gathering we don't engage with upstream llm.

Reply:
not sure how to add this tbh, prefill_enable dictate streaming